packages/main/src/themes/TextArea.css

```diff
@@ -173,19 +167,27 @@
 	max-height: 100%;
 }
 
-:host([growing-max-lines]) .ui5-textarea-mirror  {
+:host([growing-max-lines]) .ui5-textarea-mirror {
 	max-height: calc((var(--_textarea_growing_max_lines) * var(--_ui5_textarea_line_height)) * var(--sapFontSize)  + var(--_ui5_textarea_padding_top) + var(--_ui5_textarea_padding_bottom));
 }
 
-:host([rows="1"]) .ui5-textarea-inner, :host([growing-max-lines="1"]) .ui5-textarea-inner {
```

Comment:
We have to apply that min-height for the deleted selector as well host([growing-max-lines="1"]) .ui5-textarea-inner
Also, it has to be defined for host([growing-max-lines="1"]) .ui5-textarea-mirror